feature/experimental/p4rt/otg_tests/p4rt_daemon_failure_test/metadata.textproto

```diff
@@ -0,0 +1,6 @@
+# proto-file: github.com/openconfig/featureprofiles/proto/metadata.proto
+# proto-message: Metadata
+
+uuid: "78f99d29-c7db-4e6d-8284-614154efc0d1"
+plan_id: "P4RT-1.2"
+description: "P4RT Daemon Failure"
```

Comment:
Needs the testbed specified: `testbed: TESTBED_DUT_ATE_2LINKS`

Reply:
updated